### PR TITLE
Block merge of fixup commits (#75)

### LINKED ISSUE #75: Block merge of fixup commits
We don't want them in the main branch.

### PR DESCRIPTION
**Pull request by adamkudrna**
This is why:
image

Closes #75.

### CODE REVIEW

File: .github/workflows/git.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+on: [pull_request]
+
+jobs:
+  block-fixup:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2.0.0
+      - name: Block Fixup Commit Merge
+        uses: 13rac1/block-fixup-merge-action@v1.1.1
```
Comment (adamkudrna) [inline .github/workflows/git.yml:12]: This is him: https://github.com/marketplace/actions/block-fixup-commit-merge

Comment (adamkudrna) [reply .github/workflows/git.yml:12]: And source: https://github.com/13rac1/block-fixup-merge-action/blob/master/entrypoint.sh

**Comment by adamkudrna** [comment]
I've changed the action to [xt0rted/block-autosquash-commits-action](https://github.com/xt0rted/block-autosquash-commits-action) because there is a [bug in the original action](https://github.com/13rac1/block-fixup-merge-action/issues/15). This one is written in JS and runs slower (27 s vs. 11 s) but it looks more robust and it also checks for `squash!` commits.

**Comment by 13rac1** [comment]
Sorry about the delay.  https://github.com/13rac1/block-fixup-merge-action/issues/15 has been fixed in v1.1.2 if it matters.

**Pull request merged**